game/client/client_base.vpc

```diff
@@ -71,6 +46,47 @@ $Configuration
 		$AdditionalDependencies			"$BASE winmm.lib"	[$WIN32]
 		$AdditionalDependencies			"$BASE wsock32.lib Ws2_32.lib" [$BUILD_REPLAY&&$WIN32]
 	}
+
+	$Debugging
+	{
+		$Command "$OUTBINDIR\hl2.exe" [$WINDOWS]
+		$WorkingDirectory "$OUTBINDIR" [$WINDOWS]
+	}
+}
+
+$Configuration	"Debug"
+{
+	$General
+	{
+		$OutputDirectory			".\Debug_$GAMENAME" [$WINDOWS]
+		$IntermediateDirectory		".\Debug_$GAMENAME" [$WINDOWS]
+
+		$OutputDirectory			".\Debug_$GAMENAME_360"		[$X360]
+		$IntermediateDirectory		".\Debug_$GAMENAME_360"		[$X360]
+	}
+
+	$Debugging
+	{
+		$CommandArguments "-steam -game tf -insecure -novid -nojoy -nosteamcontroller -nohltv -particles 1 -noborder -particle_fallback 2 -dev -allowdebug -condebug -sw -w 800 -h 600" [$WINDOWS]
+	}
+}
+
+$Configuration	"Release"
+{
+	$General
+	{
+		// Windows generator doesn't sandbox these directories per configuration but others do :-/
+		$OutputDirectory			".\Release_$GAMENAME" [$WINDOWS]
+		$IntermediateDirectory		".\Release_$GAMENAME" [$WINDOWS]
+
+		$OutputDirectory			".\Release_$GAMENAME_360"	[$X360]
+		$IntermediateDirectory		".\Release_$GAMENAME_360"	[$X360]
+	}
+
+	$Debugging
+	{
+		$CommandArguments "-steam -game tf -insecure -novid -nojoy -nosteamcontroller -nohltv -particles 1 -noborder -particle_fallback 2 -fullscreen -w 1920 -h 1080" [$WINDOWS]
```

Comment:
Not sure what you'd want to put here, this is a mix between what you use and what I use.

Reply:
Remove -fullscreen -w 1920 -h 1080